dev/inc/SharedHelpers.h

```diff
@@ -309,6 +309,8 @@ class SharedHelpers
         }
     }
 
+    static winrt::float4 RgbaColor(winrt::Color color);
```

Comment:
>Color [](start = 42, length = 5)

@ranjeshj does this need to be const& or is winrt::Color a plain data type we can pass by value?

Reply:
Color is a struct, but we can pass it as const& similar to the winrt apis. If we can constexpr this method, then this will become a non issue.